CodeSnippetsReflection.OpenAPI.Test/PhpGeneratorTests.cs

```diff
@@ -822,4 +848,34 @@ public async Task GenerateWithCustomDateAndTimeTypes()
         Assert.Contains("->setQualityUpdatesPauseStartDate(new Date('2016-12-31'))", result);
         Assert.Contains("->setScheduledInstallTime(new Time('11:59:31.3170000'))", result);
     }
+
+    [Fact]
+    public async Task GenerateWithComplexArray()
+    {
+        const string body = @"
+          {
+          ""description"": ""Group with designated owner and members"",
+                ""displayName"": ""Operations group"",
+                ""groupTypes"": [
+                ],
+                ""mailEnabled"": false,
+                ""mailNickname"": ""operations2019"",
+                ""securityEnabled"": true,
+                ""owners@odata.bind"": [
+                     ""https://graph.microsoft.com/v1.0/users/26be1845-4119-4801-a799-aea79d09f1a2""
+                ],
+                ""members@odata.bind"": [
+                    ""https://graph.microsoft.com/v1.0/users/ff7cb387-6688-423c-8188-3da9532a73cc"",
+                    ""https://graph.microsoft.com/v1.0/users/69456242-0067-49d3-ba96-9de6f2728e14""
+                ]
+            }";
+        using var requestPayload = new HttpRequestMessage(HttpMethod.Post, $"{ServiceRootUrl}/groups")
+        {
+            Content = new StringContent(body, Encoding.UTF8, "application/json")
+        };
+        var snippetModel = new SnippetModel(requestPayload, ServiceRootUrl, await GetV1SnippetMetadata());
+        var result = _generator.GenerateCodeSnippet(snippetModel);
+        Assert.Contains("'owners@odata.bind' => [\n"+
+        "'https://graph.microsoft.com/v1.0/users/26be1845-4119-4801-a799-aea79d09f1a2', 		],", result);
```

Comment:
Do we also want to trim the whitespace after the comma as well? 
```suggestion
        "'https://graph.microsoft.com/v1.0/users/26be1845-4119-4801-a799-aea79d09f1a2',],", result);
```